src/services/expense.service.js

```diff
@@ -0,0 +1,70 @@
+const { Op } = require('sequelize');
+const { Expense } = require('../models/Expense.model.js');
+
+const getAllExpense = async ({ userId, categories, from, to }) => {
+  const params = {};
+
+  if (userId) {
+    params.userId = userId;
+  }
+
+  if (categories) {
+    params.category = categories;
```

Comment:
The 'categories' parameter is assigned directly to 'params.category'. If 'categories' is an array, this might not work as expected. Consider using Sequelize's Op.in operator to handle multiple categories, e.g., `params.category = { [Op.in]: categories };`.